scripts/be-excellent.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports = (robot) ->
+  robot.hear /\bbe\sexcellent\b/, (msg) ->
```

Comment:
Do we want `/.../i` to ignore case?

Reply:
and maybe `\s+` to allow for more whitespace

Reply:
I'll throw that and documentation in real quick and push it up

Reply:
Done and Done on the white space. I took a look at other scripts for whitespace and it seemed like static was what people went with, but I could go either way on it.